.gitignore

```diff
@@ -3,3 +3,4 @@ openwhisk-master*
 
 *.iml
 .idea
+.vscode
```

Comment:
also ignore build directory output  `target/`

Reply:
hmm!!! usually I dont do it every project as they are part of my global excludes ;).. anyways will add it

Reply:
thanks it just showed up for me on `git status`